controller_manager/src/ros2_control_node.cpp

```diff
@@ -44,6 +44,9 @@ int main(int argc, char ** argv)
 
   auto cm = std::make_shared<controller_manager::ControllerManager>(executor, manager_node_name);
 
+  const bool use_sim_time = cm->get_parameter_or("use_sim_time", false);
+  rclcpp::Rate rate(cm->get_update_rate());
```

Comment:
`rclcpp::Rate rate(cm->get_update_rate(), cm->get_clock());`
does not exist in rclcpp on humble. what happens with the following, will it use the correct clock?

Reply:
May be we can use this way from the clock?

https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1Clock.html#_CPPv4N6rclcpp5Clock9sleep_forE8DurationN7Context9SharedPtrE